### PR TITLE
Accessibility fixes

### PR DESCRIPTION
**Pull request by jancborchardt**
A bunch of accessibility fixes all over the place. Used the [WAVE Web Accessibility Tool](http://wave.webaim.org/) as usual.

Please review @nextcloud/designers @nextcloud/accessibility

### CODE REVIEW

**Review by tyrylu — APPROVED** [review]
First, thank you for those. They look good from the screen reader side of things, at least, so i have no issue approving the PR.
And just an aside, thanks for pointing the wave tool.

File: core/js/contactsmenu.js [hunk 1]
```diff
@@ -224,7 +225,7 @@
 			}));
 			this.delegateEvents();
 
-			// Show placeholder iff no avatar is available (avatar is rendered as img, not div)
+			// Show placeholder if no avatar is available (avatar is rendered as img, not div)
```
Comment (ChristophWurst) [inline core/js/contactsmenu.js]: That's not a typo: https://en.wikipedia.org/wiki/If_and_only_if 😉

File: core/templates/layout.user.php [hunk 2]
```diff
@@ -33,29 +33,21 @@
...
-						<?php p(!empty($_['application'])?$_['application']: $l->t('Apps')); ?>
-					</h1>
-					<div class="icon-caret"></div>
-				</a>
-
 				<ul id="appmenu">
 					<?php foreach ($_['navigation'] as $entry): ?>
 						<li data-id="<?php p($entry['id']); ?>" class="hidden">
 							<a href="<?php print_unescaped($entry['href']); ?>"
-							   tabindex="3"
```
Comment (ChristophWurst) [inline core/templates/layout.user.php]: Don't we need this?

File: core/js/contactsmenu.js (hunk 1 above)
Comment (skjnldsv) [reply core/js/contactsmenu.js]: TIL! 👍

Comment (jancborchardt) [reply core/js/contactsmenu.js]: Ok, it's not really necessary though cause of the following text. ;) Let's keep the confusion down maybe

File: core/templates/layout.user.php (hunk 2 above)
Comment (jancborchardt) [reply core/templates/layout.user.php]: No - try it out. ;)

**Comment by jancborchardt** [comment]
@tyrylu you're welcome! :) Any further pointing out of issues or even assistance in fixing is much appreciated! :) I will go through other apps and parts after this is merged.

Can I get another review @nextcloud/designers?

File: apps/files/templates/simplelist.php [hunk 3]
```diff
@@ -29,7 +29,7 @@
 				<a id="modified" class="columntitle" data-sort="mtime"><span><?php p($l->t( 'Modified' )); ?></span><span class="sort-indicator"></span></a>
 					<span class="selectedActions"><a href="" class="delete-selected">
 						<?php p($l->t('Delete'))?>
-						<img class="svg" alt="<?php p($l->t('Delete'))?>"
+						<img class="svg" alt=""
```
Comment (pixelipo) [inline apps/files/templates/simplelist.php]: Isn't this line supposed to be `<span class="icon icon-delete"></span>` ?

Comment (jancborchardt) [reply apps/files/templates/simplelist.php]: @pixelipo yes, however that’s a slightly separate issue and I’d like to not make this pull request bigger. ;)

Comment (jancborchardt) [reply apps/files/templates/simplelist.php]: (Btw we don’t need to repeat an extra »icon« class as we have a proper CSS rule checking for »icon-« ;)

**Comment by jancborchardt** [comment]
@Volker-E @Lartza could you review this accessibility fix too? :)

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**